helix-core/src/test/java/org/apache/helix/integration/messaging/TestP2PNoDuplicatedMessage.java

```diff
@@ -174,7 +174,10 @@ public void testP2PStateTransitionEnabled() {
       verifyP2PEnabled(startTime);
     }
 
-    Assert.assertEquals(p2pTrigged, total);
+    // Discussed with Meng who originally created this one. The success rate really depends on how
```

Comment:
Who is "Meng"? This code is for everyone to review. Please be aware of and follow the open-source code convention.

Reply:
changed to     
>// The success rate really depends on how quick participant act in relationship with controller.
   // For now, we set 90% threshold.